### ISSUE TITLE
Información de entrada Control Unit

### ISSUE THREAD

**Issue by drosselot** [IIC2343/Syllabus-2024-1#22]
Hola! No entiendo porque al Control Unit entran 20 bits si según lo que tengo entendido solo se utilizan los 8 del opcode.

**Comment by drosselot** [comment]
Perfecto! Ahora solo por curiosidad, ¿No se perdería la gracia de los opcodes? Por lo que entendí de la materia, se usaban opcodes para disminuir el número de señales que se usan, pero ahora se terminarían usando más.

**Comment by Borjampm** [comment]
Efectivamente, así es. Lo ideal sería lograr tener el opcode más chico posible. El problema es que hace bastante desafiante la implementación, por lo que los bits extra estan ahí para ayudar con eso.
